src/functions_framework/__init__.py

```diff
@@ -209,16 +239,36 @@ def create_app(target=None, source=None, signature_type=None):
     elif signature_type == "event":
         app.url_map.add(
             werkzeug.routing.Rule(
-                "/", defaults={"path": ""}, endpoint="run", methods=["POST"]
+                "/", defaults={"path": ""}, endpoint=signature_type, methods=["POST"]
             )
         )
         app.url_map.add(
-            werkzeug.routing.Rule("/<path:path>", endpoint="run", methods=["POST"])
+            werkzeug.routing.Rule(
+                "/<path:path>", endpoint=signature_type, methods=["POST"]
+            )
         )
```

Comment:
We should not modify the "event" handler. I believe this will break production, as GCF uses "/run". We should revert this.

Attempt to only modify the signature_type of cloudevent.

Reply:
my apologies, this was left over from merging master into version 1 branch I believe

Reply:
reverted to the old event if statement